src/tactic/core.lean

```diff
@@ -1612,6 +1612,24 @@ add_tactic_doc
 
 attribute [higher_order map_comp_pure] map_pure
 
+
+/-- Put a definition in the `tactic.interactive` namespace to make it usable
```

Comment:
```suggestion
/--
Copies a definition into the `tactic.interactive` namespace to make it usable
```

Could you also add the PR description here?  It would fit very nicely.